### DISCUSSION TITLE
YI Dome Camera 1080p

### DISCUSSION THREAD

**Discussion by KaHu85** [roleoroleo/yi-hack-Allwinner-v2#131] [Q&A]
Hi there,

is it possible to add this Camera - YI Dome Camera 1080p?
The Serial is BFUSY*****
I looked on to the Supported Cameras it is now updated with some more cameras but this type is not listed yet.

Many Thanks
Alex

**Comment by roleoroleo** [comment]
What's the firmware version?

**Reply by KaHu85** [reply]
4.6.0.0A_201908271549

**Reply by roleoroleo** [reply]
https://github.com/roleoroleo/yi-hack-MStar

**Reply by KaHu85** [reply]
Great!
I didnt see that.
Thx alot!
